### DISCUSSION TITLE
Pay Button add SATS denomination

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2885] [Ideas / Feature Requests]
It seems that the payment button only has some shitcoins and BTC denomination, but no SATS. :cry: 

![screenshot_20210915_192219](https://user-images.githubusercontent.com/30683012/133480211-b5547741-dee9-4a2e-baa1-eb29a8d58951.png)

**Comment by pavlenex** [comment]
It does, just type `SATS` in currency these are just examples
![Screenshot 2021-09-15 at 19 48 37](https://user-images.githubusercontent.com/36959754/133483813-d2eabda4-2840-4fb3-950f-3ce6c3dcda9b.png)

**Reply by ghost** [reply]
That's what I thought too, but that doesn't change the preview...
![screenshot_20210915_201357](https://user-images.githubusercontent.com/30683012/133487259-34ae709a-9b02-4ebc-9e5a-627ef6d652f5.png)

It's also not in the `Generated Code` block.

**Reply by pavlenex** [reply]
If you click on a button it should generate sats invoice I think. It seems we're not refreshing the preview so perhaps a bug.

**Reply by ghost** [reply]
1. Set to `100 SATS`
2. Click Button
3. See it is actually 100 SATS 
4. Go back one page
5. Now it's back to USD again.

The preview does not get refreshed.

**Reply by bolatovumar** [reply]
Made a PR here to support non-default currencies in pay button preview and generated code: https://github.com/btcpayserver/btcpayserver/pull/2896

> 5. Now it's back to USD again.

I think this is expected since this form value is hardcoded in that form field and the new value is never persisted anywhere
